### ISSUE TITLE
Can't save the qr code with the logo (only saves the qr code)

### ISSUE THREAD

**Issue by iamacatperson** [gcoro/react-qrcode-logo#10]
I wonder if you can help me. I tried saving the canvas image. However, when I save it, it doesn't have the logo in the middle, and it also doesn't have the padding (`quietZone`). It only contains the actual QR code. 

Viewing it on the browser looks exactly like how it's supposed to be (with logo and padding). It's when it's downloaded that it becomes an issue.

I am using this code to download it (on button click):

```
const refCallback = node => {
    const canvas = document.querySelector(".qr-code-wrap > canvas");
    if (node) {
      node.href = canvas.toDataURL();
      node.download = `qrcode.png`;
    }
  };
```

```
<div className="qr-code-wrap">
        <QRCode 
             value="https://github.com/gcoro/react-qrcode-logo"
             logoImage={pathToImage}
             quietZone={20}
             ecLevel="Q"
        />
</div>
```

**Comment by gcoro** [comment]
Hi, 
I tried to reproduce your issue, and for me the logo image get correctly downloaded with the QR Code - what  kind of image are you using? png? 

As for the quietZone, you're right, it gets lost in the downloaded file, may be because it is an actual padding around the canvas element and not directly draw in the canvas (this is a change I plan to make to this module when I get time)..

As a workaround, you can use a library to take a snapshot of the html, such as `html2canvas`, which should download without problems the whole QR Code, including image and quietZone. Example snippet: 
```javascript 
    html2canvas(document.querySelector('#react-qrcode-logo'))
        .then((canvas) => {
	    const link = document.createElement('a');
	    link.download = 'qrcode.png';
	    link.href = canvas.toDataURL();
	    link.click();
    });
```

**Comment by gcoro** [comment]
Hi! The padding issue should be fixed with PR #11. This fix is included in version >= 2.2.1. 

As for the logo, I don't have enough info to help... if you can provide a reproducible example I'll check itout.
